### DISCUSSION TITLE
Fix used-by feature

### DISCUSSION THREAD

**Discussion by port19x** [community/community#61728] [Repositories]
### Select Topic Area

Product Feedback

### Body

Or don't, I don't mind the free advertising :man_shrugging: 

Basically, with go get you can have dependencies from any language to any other language.
I can even make my dotfiles depend on `community/community`.

A possible fix would be to check for the same language between the lockfile and the dependency.

**Comment by ameligrana** [comment]
It should instead be fixed (e.g. by letting the possibility to disable it). See all these major programming language repositories, "Used by" sections have non-sense values

https://github.com/JuliaLang/julia
https://github.com/rust-lang/rust
https://github.com/JetBrains/kotlin
https://github.com/golang/go
https://github.com/vlang/v
https://github.com/ruby/ruby
